TotalCrossSDK/src/main/java/tc/tools/AnonymousUserData.java

```diff
@@ -86,72 +91,192 @@ private static JSONObject readJsonObject(Stream stream) throws IOException {
         }
     }
 
+    public static HttpURLConnection getConn(String url) throws IOException {
+        HttpURLConnection conn = (HttpURLConnection) new URL(url).openConnection();
+        conn.setRequestMethod("GET");
+        conn.setRequestProperty("Accept", "application/json");
+        conn.setRequestProperty("Content-Type", "application/json; charset=UTF-8");
+        conn.setDoInput(true);
+        conn.setDoOutput(true);
+        conn.setConnectTimeout(60_000);
+        conn.setReadTimeout(60_000);
+        return conn;
+    }
+
+    public static String getReponseBody(HttpURLConnection conn) throws IOException {
+        try(BufferedReader br = new BufferedReader(
+                new InputStreamReader(conn.getInputStream(), "utf-8"))) {
+            StringBuilder response = new StringBuilder();
+            String responseLine = null;
+            while ((responseLine = br.readLine()) != null) {
```

Comment:
Why use readLine and concatenate strings back when you can just read the entire content?

Reply:
As this all the requests to all endpoints have not a long response size, I agree with it.

Reply:
could give me an example of reading the entire content?